client/my-sites/plugins/marketplace/marketplace-test/index.tsx

```diff
@@ -11,40 +11,124 @@ import styled from '@emotion/styled';
  */
 import CardHeading from 'calypso/components/card-heading';
 import { Button, Card } from '@automattic/components';
-import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { getSelectedSite, getSelectedSiteId } from 'calypso/state/ui/selectors';
 import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import {
+	fetchAutomatedTransferStatus,
+	requestEligibility,
+} from 'calypso/state/automated-transfer/actions';
+import { getAutomatedTransfer } from 'calypso/state/automated-transfer/selectors';
+import QueryJetpackPlugins from 'calypso/components/data/query-jetpack-plugins';
+// import { getPlugins, isRequestingForSites } from 'calypso/state/plugins/installed/selectors';
 
 export const Container = styled.div`
-	margin: 25px;
+	margin: 0 25px;
 	padding: 25px;
 `;
 
 export default function MarketplaceTest() {
-	const selectedSite = useSelector( getSelectedSite );
+	const selectedSite = useSelector( getSelectedSite ) || {};
+	const selectedSiteId = useSelector( getSelectedSiteId );
+	// This selector is not working need to investigate why that is
+	// const pluginDetails = useSelector( ( state ) => getPlugins( state, selectedSiteId ) );
```

Comment:
I did some investigation here but couldn't figure this out yet. While the review goes on I am investigating this more intensely. If I am not able to do a non invasive fix, is it ok to add a brand new selector?